app/views/people/show.html.erb

```diff
@@ -0,0 +1,19 @@
+<%= render "shared/header" %>
+
+  <main>
+ <table class="details">
+  <tr>
+    <th>Id</th>
+    <td><%= @person.id %></td>
+  </tr>
+  <tr>
+    <th>Name</th>
+    <td><%= @person.name %></td>
+  </tr>
+</table>
+<section class="button_placement">
```

Comment:
Per CSS class naming conventions, we usually use the `-` to separate the words rather than the `_`